test/test_const_value_dist.jl

```diff
@@ -4,8 +4,43 @@ using ValueShapes
 using Test
 
 using Distributions
+using Random
 
 
 @testset "const_value_dist" begin
-    # ToDo: Add tests
+    univariate_cvd = ConstValueDist(Int64(42))
+    v = [1, 2, 3]
+    v = broadcast(Int64, v)
+    multivariate_cvd = ConstValueDist(v)
+    @test @inferred length(multivariate_cvd) == length(v)
+    
+    shape = varshape(univariate_cvd)
+
+    @test @inferred insupport(univariate_cvd, 42) == true
```

Comment:
That should be

    @test @inferred(insupport(univariate_cvd, 42)) == true

you want to test that Julia can infer the return type of `insupport`, not of the test comparison.

Reply:
Made the request change to all @inferred calls, as I believe that is the correct thing to do.